### ISSUE TITLE
Cant use service names with a period in them

### ISSUE THREAD

**Issue by cameronbraid** [berkesa/moleculer-java#7]
When creating a service in java like the following :

```
import com.lambdaworks.redis.RedisURI;

import drivenow.Environment;
import services.moleculer.ServiceBroker;
import services.moleculer.service.Action;
import services.moleculer.service.Service;
import services.moleculer.transporter.RedisSentinelTransporter;
import services.moleculer.transporter.TcpTransporter;

public class Moleculer {
	public static void main(String[] args) throws Exception {
		ServiceBroker broker = ServiceBroker.builder()
			.transporter(new TcpTransporter())
			.build();
		broker.createService(new Service("java.math") {
			public Action add = ctx -> {
				return ctx.params.get("a").asLong() +ctx.params.get("b").asLong(); 
			};
		});
		broker.start();
	}
}
```

The service is not discoverable - from node repl the service name is blank :
```
mol $ services
╔═════════════════════╤═════════╤══════════╤═════════╤════════╤═══════╗
║ Service             │ Version │    State │ Actions │ Events │ Nodes ║
╟─────────────────────┼─────────┼──────────┼─────────┼────────┼───────╢
║ $node               │       - │    OK    │       5 │      0 │ 2     ║
║                     │       - │    OK    │       0 │      0 │ 1     ║
╚═════════════════════╧═════════╧══════════╧═════════╧════════╧═══════╝
```

**Comment by berkesa** [comment]
So there are many possible reasons for this. TCP transporter uses multicast/broadcast UDP discovery by default. If the two nodes (NodeJS and Java nodes) are behind a firewall (or on WAN), they may not see each other. If you run these nodes on a single machine, then the problem may be different. Could you attach the log files? (The first half minute are enough.)

**Comment by berkesa** [comment]
Now I just see that the java node is connected, only the service can not be seen because '.' in its name. Thanks, so it is not necessary to send the log. I'll try it out soon.....

**Comment by berkesa** [comment]
I found the problem, and fixed. In the next version you will be able to use a dot character in the name of the Service. It'll be released a few days later.

**Comment by berkesa** [comment]
Fixed in 1.0.7
